region/crateria/central/Parlor and Alcatraz.json

```diff
@@ -1122,24 +1146,98 @@
+      "devNote": [
+        "Morph could also be used instead of Grapple, but with Morph there is an easier strat that goes up through the tunnel to the left."
+      ]
+    },
+    {
+      "link": [5, 8],
+      "name": "HiJump Ceiling Clip Alcatraz Escape",
+      "notable": true,
+      "requires": [
+        "HiJump",
+        "h_ZebesIsAwake",
+        {"or": [
+          "canBeVeryPatient",
+          {"ammo": {"type": "Super", "count": 1}}
+        ]},
+        "canPreciseCeilingClip",
+        "canTrickyUseFrozenEnemies",
+        {"enemyDamage": {
+          "enemy": "Geemer (blue)",
+          "type": "contact",
+          "hits": 1
+        }},
+        "canInsaneJump"
+      ],
+      "note": [
+        "Wait for a global Geemer to make the long trip along the map, or shoot a Super 20 to 30 seconds after entering the room to knock it off the ceiling and save a lot of time.",
+        "Hold right against the bomb blocks, take damage from the Geemer, jump, aim-down, and shoot the Geemer shortly before landing.",
+        "Samus should end up standing on the ground with the Geemer frozen on the wall slightly above the ground (overlapping Samus' hitbox).",
+        "Facing the bomb blocks and pressed against them, jump while holding down (but not crouched) and then press forward at a precise time to break the aim-down pose.",
+        "If successful, Samus will clip up through the ceiling.",
+        "This can be attempted several times before the Geemer thaws."
+      ]
+    },
+    {
+      "link": [5, 8],
+      "name": "G-Mode Forced Standup Alcatraz Escape",
+      "notable": true,
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_ZebesIsAwake",
+        "canBeVeryPatient",
+        {"ammo": {"type": "Super", "count": 3}},
+        "h_canArtificialMorphCrystalFlash",
+        "canPauseAbuse",
+        {"autoReserveTrigger": {}},
+        {"enemyDamage": {
+          "enemy": "Geemer (blue)",
+          "type": "contact",
+          "hits": 1
+        }}
+      ],
+      "note": [
+        "Enter with direct G-Mode artificial morph, and perform a Crystal Flash to refill reserve energy.",
+        "Do this all the way to the left, to prevent the Geemer from being destroyed by the lingering light orb.",
+        "Wait for a global Geemer to make the long trip along the map, or shoot a Super 20 to 30 seconds after entering the room to knock it off the ceiling and save a lot of time.",
+        "Either way, after the Geemer comes down into Alcatraz, shoot a Super to knock it off the ledge to make it bypass the light orb.",
+        "Damage down until one Geemer hit away from running out of energy.",
+        "If Samus has an ETank, it will take longer to damage down, in which case an additional Super can be used to knock the Geemer onto the floating platform, to allow unlimited time for damaging down.",
```

Comment:
```suggestion
        "If Samus has at least one Energy Tank, it will take longer to damage down, in which case an additional Super can be used to knock the Geemer onto the floating platform, to allow unlimited time for damaging down.",
```